packages/api-tests/test/test-helpers.js

```diff
@@ -12,8 +12,9 @@ const selectWindow = (client, handle) => {
 const openNewWindow = (client, options) => {
   const script = (options, callback) => {
     ssf.app.ready().then(() => {
-      new ssf.Window(options);
-      setTimeout(() => callback(), 500);
+      new ssf.Window(options, () => {
+        setTimeout(() => callback(), 500);
```

Comment:
If the window success callback is being used (which I think is a good thing), can the setTimout be removed?